app/models.py

```diff
@@ -2988,5 +2996,32 @@ def extend_subscription(
             },
         )
 
+    @classmethod
+    def downloaded_provider_complaint(cls, admin_user_id: int, complaint_id: int):
+        cls.create(
+            admin_user_id=admin_user_id,
+            action=AuditLogActionEnum.download_provider_complaint.value,
+            model="ProviderComplaint",
+            model_id=complaint_id,
+            data={},
+        )
+
+
+class ProviderComplaintState(EnumE):
+    new = 0
+    reviewed = 1
 
-# endregion
```

Comment:
should we move this `endregion` to the top? It corresponds to the Phone region I think.

Reply:
:+1: